.clang-tidy

```diff
@@ -0,0 +1,4 @@
+Checks: '-*,performance-*,modernize-*,-modernize-use-trailing-return-type,-modernize-use-nodiscard,-modernize-avoid-c-arrays,-modernize-use-auto'
```

Comment:
does this line enable or disable checks ?

Reply:
When it starts with `-` it disables the checks. So `-*` means disable everything, then enable all `performance` and `modernize` checks, then disable specific checks in the two enabled groups.

Reply:
thanks for the explanation